src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/TypeConverter.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.companieshouse.overseasentitiesapi.utils;
+
+import uk.gov.companieshouse.api.model.company.RegisteredOfficeAddressApi;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.AddressDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.Address;
+
+public class TypeConverter {
+    private TypeConverter() {
+        throw new IllegalAccessError("Use the static methods instead of instantiating Converters");
+    }
```

Comment:
We could test this using Reflection? Not sure what the CH policy is on using reflection for testing but in this case I'd imagine its little overhead to bump the coverage up.